base/Dockerfile

```diff
@@ -85,6 +85,9 @@ RUN apt-get update && \
     rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/* && \
     update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 
+RUN wget -nv -O /bin/hadolint https://github.com/hadolint/hadolint/releases/download/v2.10.0/hadolint-Linux-x86_64 \
```

Comment:
The existing Docker file is architecture neutral so we can build on/for Arm hosts.  hadolint does provide architecture specific binaries.  Please update the line to pull the appropriate architecture based on Docker variables.

Reply:
In the same file we have an example with `rustup` where we handle both `x86_64` and `aarch64` and we also check the hash.

This binary is only relevant for CI, this should not be in the `base` image. Haloband could be installed as part of the GitHub Actions workflow, or it could be added only in the `ci` (possibly `localci`) images.

Also, `veracruz-docker-image` has now been moved into `veracruz` (veracruz-project/veracruz#411), so this PR should be closed and re-opened on `veracruz`